executor/aggregate.go

```diff
@@ -614,12 +630,17 @@ func (e *HashAggExec) fetchChildData(ctx context.Context) {
 		if chk.NumRows() == 0 {
 			return
 		}
+		e.memTracker.Consume(chk.MemoryUsage())
```

Comment:
How about calculating the difference of memory usage before and after `Next`, and then we can only invoke `memTracker.Consume` once.